main/org.thingpedia.iot.climate/manifest.tt

```diff
@@ -121,19 +121,19 @@ abstract class @org.thingpedia.iot.climate
   #_[confirmation="set your fan mode to $mode"]
   #[confirm=true];
 
-  monitorable query swing_mode(out mode: Enum(on,off,auto,low,medium,high,middle,focus,diffuse))
+  monitorable query swing_mode(out mode: Enum(swing_off,swing_vertical,swing_horizontal,swing_both))
```

Comment:
Why are we duplicating the word "swing" in the enum?

Reply:
https://developers.home-assistant.io/docs/core/entity/climate#swing-modes

Reply:
We can clean the enum value in our JS code. Using good words for the enum name is more important than matching the Home Assistant API exactly because it affects the quality of the natural language understanding.